torchtune/modules/common_utils.py

```diff
@@ -48,3 +54,118 @@ def reparametrize_as_dtype_state_dict_post_hook(
             state_dict[k] = v.to(dtype)
             if offload_to_cpu:
                 state_dict[k] = state_dict[k].cpu()
+
+
+# mmap.MAP_SHARED is not supported on Windows but this change targets colab.
+if torch.__version__ >= "2.5.0.dev20240906" and not sys.platform == "win32":
```

Comment:
Just a sanity check here: is this if/else just to ensure that no one tries to directly import the `_low_ram_reparametrize_as_dtype_state_dict_post_hook` API on an unsupported environment? Mainly asking because we have the equivalent checks in `_register_reparametrize_state_dict_hooks` now

Reply:
You're right, removing this if else